app/views/sephora_style_guide/home/_typography.slim

```diff
@@ -25,30 +25,40 @@ article#typography
           td
             p.text-sephora-book v-bind:style="{fontSize: fontSize + 'px'}"
               ' {{ text }}
+          td
+            p.text-sephora-book style='font-size: 18px;' .text-sephora-book
         tr
           td
             p.text-sephora-italic style='font-size: 18px;' Sephora Serif Book Italic
           td
             p.text-sephora-italic v-bind:style="{fontSize: fontSize + 'px'}"
               ' {{ text }}
+          td
+            p.text-sephora-italic style='font-size: 18px;' .text-sephora-italic
         tr
           td
             p.text-avalon-bold style='font-size: 18px;' Avalon Bold
           td
             p.text-avalon-bold v-bind:style="{fontSize: fontSize + 'px'}"
               ' {{ text }}
+          td
+            p.text-avalon-bold style='font-size: 18px;' .text-avalon-bold
         tr
           td
             p.text-avalon-demi style='font-size: 18px;' Avalon Demi
           td
             p.text-avalon-demi v-bind:style="{fontSize: fontSize + 'px'}"
               ' {{ text }}
+          td
+            p.text-avalon-demi style='font-size: 18px;' .text-avalon-demi
```

Comment:
possible to add the style to some class? Also the classes need not be in the exact font that they represent. And would be nice to show the `include` mixins when added to CSS.